### PR TITLE
MNT Travis shared config

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: client/src/images/securityKey.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80 80"><defs><style>.a{fill:#d6e1ea;}.b{fill:#333a48;}.c{fill:#ffcf6e;}</style></defs><title>U2F option</title><circle class="a" cx="40" cy="40" r="40"/><path class="b" d="M22.65,50.17a1.24,1.24,0,0,0,1.24,1.25H60.54a3.88,3.88,0,0,0,3.67-4.07V30.54a3.88,3.88,0,0,0-3.67-4.06H23.89a1.24,1.24,0,0,0-1.24,1.24ZM58,36.46a2.29,2.29,0,1,1-2.33,2.28A2.31,2.31,0,0,1,58,36.46Z"/><path class="b" d="M8.51,32.71H24.31a0,0,0,0,1,0,0v9.47a3,3,0,0,1-3,3H11.51a3,3,0,0,1-3-3V32.71A0,0,0,0,1,8.51,32.71Z" transform="translate(55.36 22.54) rotate(90)"/><circle class="c" cx="42.23" cy="39.23" r="7.48"/><rect class="c" x="16.2" y="38.74" width="2.08" height="10.81" transform="translate(61.38 26.9) rotate(90)"/><rect class="c" x="17.03" y="36.25" width="2.08" height="9.14" transform="translate(58.89 22.74) rotate(90)"/><rect class="c" x="16.2" y="28.35" width="2.08" height="10.81" transform="translate(50.99 16.51) rotate(90)"/><rect class="c" x="17.03" y="32.5" width="2.08" height="9.14" transform="translate(55.15 19) rotate(90)"/></svg>
```
Comment (maxime-rainville) [inline client/src/images/securityKey.svg:1]: Can you provide some clarification about what this SVG file is about?

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
There seems to be other changes beyond the travis file in here.

File: client/src/images/securityKey.svg (hunk 1 above)
Comment (emteknetnz) [reply client/src/images/securityKey.svg:1]: The NPM_TEST of the shared config will [delete the dist folder](https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/base.yml#L70), [build](https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/base.yml#L78) and [diff](https://github.com/silverstripe/silverstripe-travis-shared/blob/master/config/base.yml#L111) files.  

In this instance this SVG was added only to the dist folder, and not the src folder, so it was being deleted for the dist folder but not recreated.  I fixed this by adding it to the src folder, and tweaking webpack to copy it over

**Review by maxime-rainville — APPROVED** [review]
All good

**Pull request merged**